docs/standards.md

```diff
@@ -0,0 +1,136 @@
+     until it is ready.
+
+2. Assign the product, design, and engineering leads as reviewers, as
+   appropriate.
+
+   - The area leads will scan the PR to determine who should review it. They may
+     also review it themselves.
+
+   - If you know who needs to review your PR needs, feel free to assign those
+     people directly instead of assigning the leads.
+
+3. If the review needs a meeting or discussion time, add it to the
+   [discussion topics](https://docs.google.com/spreadsheets/d/1wd9WsmNHdLXl-smL_A63oBDRIkALJTELXcITl4iTdMs/edit#gid=1456748459)
+   and we'll pick it up at the end of a standup. If it lingers more than a
+   couple of days, ping an area lead.
+
+4. If reviewers have questions or see things that need to be fixed or changed,
+   they should leave those questions or comments in the review. Do not mark the
+   PR as "approved" unless you really mean it.
+
+   - It is okay to mark a PR as approved if you only want a small change or have
+     a minor question and don't want the PR author to have to wait for you to
+     review it again later. We are allowed to trust each other if we want to.
+     But for bigger changes, mark the PR as "changes requested" or "comment."
+
+   - If a PR uncovers work _outside the scope of the PR_ that needs to be done,
+     document that with new issues. Refer to the PR in the new issue.
+
+     - If the new work is technical debt or a bug, place it directly into the
+       backlog instead of the project inbox and label it accordingly.
+
+5. When all requested reviewers have approved a PR (and the required automated
+   checks have completed successfully), it can be merged. **PRs should be merged
+   by their authors**, since they have the most context about it and can resolve
+   any merge conflicts that might arise.
+
+   - PR authors are free to enable auto-merge on their PRs and GitHub will
+     automatically merge it once all of the requirements below are satisfied.
+
+   - We merge using the squash+rebase strategy in GitHub. The GitHub behavior is
+     something like this:
+
+     ```sh
+     git checkout main
+     git merge --squash <branch>
+     git push
+     ```
+
+     > [!NOTE]  
+     > You cannot actually do it manually in our repo as the `main` branch does
+     > not allow remote pushes.
+
+```mermaid
+%%{ init: { "flowchart": { "curve": "linear" }}}%%
+graph TD;
+  A[Open pull<br>request]-->B{Ready to<br>review?};
+  B--no-->A
+  B--yes-->C{Do you know<br>who reviews?}
+  C--no-->D[Assign leads<br>to review]
+  D-->E[Assign reviewers]
+  C--yes-->E
+  E-->F{Reviewers<br>approved?}
+  F--no-->G[Make changes]-->B
+  F--yes-->H[Merge]
+```
+
+#### Branch protections
+
+PRs cannot be merged into `main` unless they satisfy our branch protection
+rules:
+
+- All PRs must be reviewed and approved by at least one other person.
+- All PRs must pass all required automated checks. The list of required checks
```

Comment:
All #noqa must be documented or defended in the PR? There are definitely going to be false flags in tests and times we'll want to ignore things.

Reply:
Good catch. I had to google "noqa" but I get it now. 😆 Will update.